app/js/questions.js

```diff
@@ -1,5 +1,4 @@
 const flaggedQuestions = {}
```

Comment:
Oh lord why is this a const when it's clearing being changed later in the code (I know it wasn't you!)

Can this be changed to a var without issues?